tox.ini

```diff
@@ -171,27 +168,27 @@ envlist =
     {py3.8,py3.11,py3.12}-quart-latest
 
     # Redis
-    {py2.7,py3.7,py3.8}-redis-v{3}
+    {py3.7,py3.8}-redis-v{3}
     {py3.7,py3.8,py3.11}-redis-v{4}
     {py3.7,py3.11,py3.12}-redis-v{5}
     {py3.7,py3.11,py3.12}-redis-latest
 
     # Redis Cluster
-    {py2.7,py3.7,py3.8}-rediscluster-v{1,2}
+    {py3.7,py3.8}-rediscluster-v{1,2}
```

Comment:
3.6?

Reply:
Changed to `{py3.6,py3.8}-rediscluster-v{1,2}` (same reasoning as above)